telescope/functions/autodiscover.py

```diff
@@ -51,25 +53,44 @@ def get_kubernetes_uniqueness(r: "kubernetes.clients.models.V1Pod"):
 
 def kube_autodiscover(label_selector, **kwargs) -> List[Dict[str, str]]:
     """:returns List of Tuple containing - pod name, pod namespace, container name"""
-    # noinspection PyUnresolvedReferences
-    if kube_client is not None:
-        seen_uniqueness = set()
-        results = []
+    seen_uniqueness = set()
+    results = []
+    if os.getenv("TELESCOPE_KUBERNETES_METHOD", "") == "kubectl":
+        jp = (
+            """'{range .items[*]}{.metadata.name}{","}{.metadata.namespace}{","}{.metadata.generate_name}{"||"}{end}'"""
+        )
+        cmd = "kubectl get pod -l " + label_selector + " -A -o=jsonpath=" + jp
```

Comment:
would be cleaner to make this an f string

Reply:
agreed - I had some `{`'s in the `jp` above, which is why I originally didn't